ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/network/DefaultP2PNetwork.java

```diff
@@ -347,11 +344,16 @@ DNSDaemonListener createDaemonListener() {
                 .build();
         final DiscoveryPeer peer = DiscoveryPeer.fromEnode(enodeURL);
         peers.add(peer);
-        rlpxAgent.connect(peer);
```

Comment:
why are we no longer trying to connect to the peer?

Reply:
you put this back in, right @pinges

Reply:
It looks like it is just moved to the subsequent block on line 357 after adding to the peer table